docs/source/dev/test_notes.rst

```diff
@@ -80,12 +80,12 @@ tested. The tests look something like
             assert_almost_equal(self.res1.foo, self.res2.foo, 4)
 
     if __name__ == "__main__":
```

Comment:
I think we should remove this from the documentation know.
We don't want any `if __name__ == "__main__"` parts anymore.

Adding a section about running individual test(classes) on the command line would be better and reflect more modern usage.
However, that can be a separate PR. This PR is for the transition to pytest